site/AntDesign.Docs/Demos/Components/Tag/demo/Colorful.razor

```diff
@@ -4,35 +4,44 @@
     </div>
     <br />
     <div>
-        <Tag Color="magenta">magenta</Tag>
-        <Tag Color="pink">pink</Tag>
-        <Tag Color="red">red</Tag>
-        <Tag Color="volcano">volcano</Tag>
-        <Tag Color="orange">orange</Tag>
-        <Tag Color="green">green</Tag>
-        <Tag Color="cyan">cyan</Tag>
-        <Tag Color="blue">blue</Tag>
-        <Tag Color="lime">lime</Tag>
-        <Tag Color="geekblue">geekblue</Tag>
-        <Tag Color="purple">purple</Tag>
+        <Tag PresetColor="@PresetColor.Magenta">magenta</Tag>
+        <Tag PresetColor="@PresetColor.Pink">pink</Tag>
+        <Tag PresetColor="@PresetColor.Red">red</Tag>
+        <Tag PresetColor="@PresetColor.Volcano">volcano</Tag>
+        <Tag PresetColor="@PresetColor.Orange">orange</Tag>
+        <Tag PresetColor="@PresetColor.Green">green</Tag>
+        <Tag PresetColor="@PresetColor.Cyan">cyan</Tag>
+        <Tag PresetColor="@PresetColor.Blue">blue</Tag>
+        <Tag PresetColor="@PresetColor.Lime">lime</Tag>
+        <Tag PresetColor="@PresetColor.GeekBlue">geekblue</Tag>
+        <Tag PresetColor="@PresetColor.Purple">purple</Tag>
+        <Tag PresetColor="@PresetColor.Yellow">yellow</Tag>
+        <Tag PresetColor="@PresetColor.Gold">gold</Tag>
     </div>
     <br />
     <div>
         Inverse:
     </div>
     <br />
     <div>
-        <Tag Icon="taobao" Color="blue-inverse">blue-inverse</Tag>
-        <Tag Color="orange-inverse">orange-inverse</Tag>
-        <Tag Icon="skype" Color="red-inverse">red-inverse</Tag>
-        <Tag Icon="weibo" Color="purple-inverse">purple-inverse</Tag>
+        <Tag Icon="taobao" Color="@("blue-inverse")">blue-inverse</Tag>
```

Comment:
There are no need to use @("") I think.

Reply:
This is true - forgot to remove it when changing Color from OnOf to just a string. Will fix